<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LivePlotWindow</class>
 <widget class="QMainWindow" name="LivePlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Live Chart</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/chart</normaloff>:/icons/chart</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="LivePlot" name="plot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Standard</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Graph"/>
   <addaction name="actionRemove_Graph"/>
   <addaction name="separator"/>
   <addaction name="actionOptions"/>
   <addaction name="actionTitle"/>
   <addaction name="actionLegend"/>
   <addaction name="separator"/>
   <addaction name="actionCrosshair"/>
   <addaction name="actionTrack_Cursor"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionCopy"/>
  </widget>
  <action name="actionAdd_Graph">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/chart-add</normaloff>:/icons/chart-add</iconset>
   </property>
   <property name="text">
    <string>Add Graph</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/copy</normaloff>:/icons/copy</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionRemove_Graph">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/chart-remove</normaloff>:/icons/chart-remove</iconset>
   </property>
   <property name="text">
    <string>Remove Graph</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionTitle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/title</normaloff>:/icons/title</iconset>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/legend</normaloff>:/icons/legend</iconset>
   </property>
   <property name="text">
    <string>Legend</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/options</normaloff>:/icons/options</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCrosshair">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/crosshair</normaloff>:/icons/crosshair</iconset>
   </property>
   <property name="text">
    <string>Crosshair</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionTrack_Cursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/cursor-tag</normaloff>:/icons/cursor-tag</iconset>
   </property>
   <property name="text">
    <string>Track Cursor</string>
   </property>
   <property name="toolTip">
    <string>Highlights all points in line with the mouse cursor and shows their positions in the axis</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LivePlot</class>
   <extends>QWidget</extends>
   <header>charts/liveplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
